crates/compiler/parse/src/keyword.rs

```diff
@@ -7,5 +7,6 @@ pub const IS: &str = "is";
 pub const DBG: &str = "dbg";
 pub const EXPECT: &str = "expect";
 pub const EXPECT_FX: &str = "expect-fx";
+pub const CRASH: &str = "crash";
 
-pub const KEYWORDS: [&str; 8] = [IF, THEN, ELSE, WHEN, AS, IS, EXPECT, EXPECT_FX];
+pub const KEYWORDS: [&str; 9] = [IF, THEN, ELSE, WHEN, AS, IS, EXPECT, EXPECT_FX, CRASH];
```

Comment:
I think `DBG` was not added to `KEYWORDS` by mistake?

Reply:
Huh, surprised this didn't cause a merge conflict! Fixed in 71fc5445367c23e2c79bf56b634ccdb0c030f414